### ISSUE TITLE
TexMate 2

### ISSUE THREAD

**Issue by btywoniuk** [PowerShell/EditorSyntax#181]
Pleass add support for TexMate 2 (.tmbundle)

**Comment by msftrncs** [comment]
I don't think it is in the interest of this repository to add support for TextMate directly as a tmBundle.   Instead, this repository can serve as the source of the tmLanguage file of a tmBundle, which can then be maintained by someone that has a more direct connection to the TextMate environment.  That is how support for PowerShell is provided for other editors such as Atom or Sublime.

Unfortunately it doesn't appear that anyone has done that yet for TextMate 2.
